fivetran_provider_async/example_dags/example_fivetran_bigquery.py

```diff
@@ -49,13 +49,15 @@
         task_id="fivetran-task",
         fivetran_conn_id="fivetran_default",
         connector_id="{{ var.value.connector_id }}",
+        deferrable=False,
```

Comment:
Outside of the `example_fivetran.py` DAG, every operator/sensor now has `deferrable=False`. This makes me think we are nudging folks to use the non-deferrable functionality rather than using it. I assume our opinion is that deferrable is a better practice when using this version of the provider. Maybe I'm reading too much into this, but we would definitely have more examples of _not_ using deferrable functionality with these updates.

Reply:
good point. I have them to use deferrable. thanks for pointing pointing out